src/node_http2.cc

```diff
@@ -89,7 +89,7 @@ ssize_t Http2Session::OnCallbackPadding(size_t frameLen,
   Context::Scope context_scope(context);
 
 #if defined(DEBUG) && DEBUG
-  CHECK(object->Has(context, env()->ongetpadding_string()).FromJust());
+  // CHECK(object->Has(context, env()->ongetpadding_string()).FromJust());
```

Comment:
This should be

```js
CHECK(object()->Has(context, env()->ongetpadding_string()).FromJust());
```

Reply:
Thanks, I was missing where object was coming from. Will update shortly.